.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/eDocCabinet
+    docker:
+      - image: circleci/node:4.8.2
+      - image: mongo:3.4.4
```

Comment:
Taking a stub at CircleCI 2.0 is pretty awesome! Consider using a postgres image over the mongo one.